lib/trino-collect/src/test/java/io/trino/collect/cache/TestEvictableCache.java

```diff
@@ -542,4 +546,46 @@ public void testInvalidateAndLoadConcurrently(Invalidation invalidation)
             executor.awaitTermination(10, SECONDS);
         }
     }
+
+    @Test(dataProvider = "disabledCacheImplementations")
+    public void testPutOnEmptyCache(DisabledCacheImplementation disabledCacheImplementation)
+    {
+        Map<Object, Object> cache = EvictableCacheBuilder.newBuilder()
+                .maximumSize(0)
+                .disabledCacheImplementation(disabledCacheImplementation)
+                .build()
+                .asMap();
+
+        int key = 0;
+        int value = 1;
+        assertThat(cache.put(key, value)).isNull();
+        assertThat(cache.put(key, value)).isNull();
+        assertThat(cache.putIfAbsent(key, value)).isNull();
+        assertThat(cache.putIfAbsent(key, value)).isNull();
+    }
+
+    @Test
+    public void testPutOnNonEmptyCache()
```

Comment:
The cache under test is empty.
Add a comment capturing https://github.com/trinodb/trino/pull/14702#discussion_r1003188017

Reply:
I prefer to rename the test